charts/nfs-subdir-external-provisioner/templates/clusterrole.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     release: {{ .Release.Name }}
   name: {{ template "nfs-subdir-external-provisioner.fullname" . }}-runner
 rules:
+  - apiGroups: [""]
```

Comment:
same changes required in the [`deploy/`](https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blob/master/deploy/objects/clusterrole.yaml) as well

Reply:
@khalsa-ji
bump